trust-store-trust-policy-specification.md

```diff
@@ -71,7 +71,7 @@ Trust policy for a simple scenario where ACME Rockets uses only artifacts signed
             "name": "wabbit-networks-images",
             "registryScopes": [ "*" ],
             "signatureVerification" : "audit",
-            "trustStore": "ca:acme-rockets",
+            "trustStores": ["ca:acme-rockets"],
```

Comment:
Good to see it's an array, but should we provide an example of two `["ca:acme-rockets-staging", "ca:acme-rockets-prod"]`